CLI/turbot.py

```diff
@@ -136,28 +139,67 @@ def parse_filename(file_path):
     prefix = filename.split('.')[0]
     return prefix
 
+def isValidDestination(file_path):
+    print(file_path)
+    if not file_path[-1] == '/':
+        if os.path.isdir(file_path.split('/')[:-2].join('/')):
```

Comment:
There are other ways to specify a directory other than a trailing slash. Namely using `.` wildcards, which would break this.
I'd also recommend doing a `str.rfind('/')` to find the last index and slicing with that, rather than splitting and joining